Index: sandbox/src/main/java/Mfp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Mfp {\r\n    public static void main (String[] args) {\r\nSystem.out.println(\"hello world!\");\r\n    }\r\n};\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/sandbox/src/main/java/Mfp.java b/sandbox/src/main/java/Mfp.java
--- a/sandbox/src/main/java/Mfp.java	(revision efd709c70240ffcb7248f8b9ddcdf174fee8d604)
+++ b/sandbox/src/main/java/Mfp.java	(date 1656708755143)
@@ -2,5 +2,9 @@
     public static void main (String[] args) {
 System.out.println("hello world!");
     }
-};
+    public static void hello() {
+
+    }
+}
+;
 
