Index: sandbox/src/main/java/Mfp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Mfp {\r\n    public static void main (String[] args) {\r\nSystem.out.println(\"hello world!\");\r\n    }\r\n};\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/sandbox/src/main/java/Mfp.java b/sandbox/src/main/java/Mfp.java
--- a/sandbox/src/main/java/Mfp.java	(revision efd709c70240ffcb7248f8b9ddcdf174fee8d604)
+++ b/sandbox/src/main/java/Mfp.java	(date 1656708894443)
@@ -1,6 +1,9 @@
 public class Mfp {
     public static void main (String[] args) {
 System.out.println("hello world!");
+    }
+    public static void hello() {
+
     }
 };
 
Index: .idea/gradle.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"GradleSettings\">\r\n    <option name=\"linkedExternalProjectsSettings\">\r\n      <GradleProjectSettings>\r\n        <option name=\"delegatedBuild\" value=\"true\" />\r\n        <option name=\"testRunner\" value=\"GRADLE\" />\r\n        <option name=\"distributionType\" value=\"DEFAULT_WRAPPED\" />\r\n        <option name=\"externalProjectPath\" value=\"$PROJECT_DIR$/sandbox\" />\r\n        <option name=\"gradleJvm\" value=\"18\" />\r\n        <option name=\"modules\">\r\n          <set>\r\n            <option value=\"$PROJECT_DIR$/sandbox\" />\r\n          </set>\r\n        </option>\r\n      </GradleProjectSettings>\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/gradle.xml b/.idea/gradle.xml
--- a/.idea/gradle.xml	(revision efd709c70240ffcb7248f8b9ddcdf174fee8d604)
+++ b/.idea/gradle.xml	(date 1656628545621)
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
